src/index.js

```diff
@@ -0,0 +1,153 @@
+'use strict';
+
+// Selects the HTML Elements the events will occur on
+const increaseTempButton = document.querySelector('#increaseTempControl');
+const decreaseTempButton = document.querySelector('#decreaseTempControl');
+const tempValue = document.querySelector('#tempValue');
+const landscape = document.querySelector('#landscape');
+const headerCityName = document.querySelector('#headerCityName');
+const currentTempButton = document.querySelector('#currentTempButton');
+const cityNameInput = document.querySelector('#cityNameInput');
+const BASE_URL = 'http://127.0.0.1:5000';
+
+// Makes functions to run when events occur
+const state = {
+	city: 'South Lake Tahoe',
+	lat: 38.9332411,
+	lon: -119.9843482,
+	temp: 60,
+};
+
+const convertKtoF = (temp) => {
+	return (temp - 273.15) * (9 / 5) + 32;
+};
+
+const getLatAndLong = () => {
+	axios
+		.get(`${BASE_URL}/location`, {
+			params: {
+				// key: process.env['LOCATION_KEY'],
+				// q: 'Seattle, Washington, USA',
+				q: state.city,
+			},
+		})
+		.then((response) => {
+			// console.log('success!', response.data);
```

Comment:
Let's get rid of any code we aren't using anymore
```suggestion
			params: {
				q: state.city,
			},
		})
		.then((response) => {
```